src/main.rs

```diff
@@ -107,7 +107,7 @@ fn print_bench_time(benchmarking: bool, benchmark_warn: Option<u128>, name: &str
 
     // This is different to module bench times fyi
     let t: Duration = time.unwrap().elapsed();
-    let mut t_output: String = format!("{:2?}", t);
+    let mut t_output: String = format!("{:.3} ms", t.as_micros() as f64 / 1000.);
```

Comment:
Not a fan of this change - would rather have more accuracy than easier units, even if it's at a scale that it hardly matters

Reply:
This is for comparation. You can find the bottleneck at the first glance if the all have the same unit.

Reply:
Add it as a command line argument and let the user decide then, don't force it though